redfish-core/lib/update_service.hpp

```diff
@@ -19,6 +19,7 @@
 
 #include <app.hpp>
 #include <boost/container/flat_map.hpp>
+#include <boost/range/adaptor/reversed.hpp>
```

Comment:
Why do we need this?

Reply:
I guess you are starting at the back, you are looking for the last Logging Entry ?

Reply:
I was using it to iterate through the log errors backwards since the code update error addition is relatively recent, but it's not necessary and can be taken out if need be

Reply:
it's usually the fourth or fifth from the end

Reply:
Ack.